src/saving/SaveHelper.cs

```diff
@@ -38,6 +40,18 @@ public enum SaveOrder
         FileSystem,
     }
 
+    /// <summary>
+    ///   Checks whether the last save is made within a timespan of set duration.
+    /// </summary>
+    /// <remarks>
+    ///   <para>
+    ///     Used for knowing whether to show confirmation dialog on the pause menu when exiting the game.
+    ///     TODO: Implement a method overriding this flag as old in some way.
+    ///   </para>
+    /// </remarks>
+    /// <returns>True if the last save is still recent, false if otherwise.</returns>
+    public static bool SavedRecently => DateTime.Now - lastSave < Constants.RecentSaveTime;
```

Comment:
Doesn't this need a null check?

Reply:
Probably, it works on first save attempt though...
Should `GetValueOrDefault` be used or should ternary be used here?
```csharp
DateTime.Now - lastSave.GetValueOrDefault() < Constants.RecentSaveTime;
```

Reply:
I would not use GetValueOrDefault here

Reply:
Alright.